deploy-manage/monitor/autoops/search-tier-view-autoops-serverless.md

```diff
@@ -0,0 +1,96 @@
+---
+applies_to:
+  serverless:
+navigation_title: Search Tier view
+---
+
+# Search Tier view in AutoOps for Serverless
+
+The Search Tier view in AutoOps for Serverless provides visibility into the consumption of search VCUs, which are a type of [compute billing dimension](/deploy-manage/monitor/autoops/autoops-for-serverless.md#compute-billing-dimensions). This view helps you understand how search activities and performance contribute to your search VCU consumption and as a result, your project's bill. 
+
+This view provides both high-level project summaries and detailed index-level and data stream-level breakdowns. 
+
+To get to the Search Tier view, [access AutoOps](/deploy-manage/monitor/autoops/access-autoops-for-serverless.md) in your project and then select **Search Tier** from the navigation menu.
+
+## Project-level insights
+
+On the **Search Tier** page, the top half of the screen offers general insights at the project level.
+
+:::{image} /deploy-manage/images/search-tier-project-level-features.png
+:screenshot:
+:alt: Screenshot showing the features in the top half of the Search Tier page
+:::
+
+Use the following features to explore this view:
+* Use the built-in **project picker** to switch between projects. This allows you to make quick context changes without needing to navigate back to your {{ecloud}} home page to select a different project.
+* Select **custom time windows** to explore usage and performance data up to the last 10 days. The charted data is bucketed per day except when you select a period of up to 72 hours, when it is bucketed per hour.
+* Explore different **visualizations** presenting the trend of search VCU usage over time and how it compares to the performance of the search tier in terms of search rate and latency.
+* View the **annotations** overlaying the search VCUs usage chart to understand when the search power and boost window changed during the selected time period and how that might have affected the autoscaling of your project (and consequently your VCU consumption). 
+* Gain insights from the **performance charts** depicting search rate and search latency trends to understand why your VCU consumption might fluctuate over time. 
+
+## Index and data stream-level insights
+ 
+The bottom half of the screen offers a more granular breakdown table of index-level and data stream-level insights into search performance. 
+
+:::{image} /deploy-manage/images/search-tier-breakdown-table.png
+:screenshot:
+:alt: Screenshot showing an expanded row in the data set table in the bottom half of the Search Tier page
+:::
+
+Each row of the table represents a single index or data stream and provides the following information:
```

Comment:
```suggestion
Each row of the table represents a single index or data stream, providing the following information:
```
(just to avoid both 'and' and 'or' in the same clause.